frontend/src/component/changeRequest/changeRequest.types.ts

```diff
@@ -100,7 +100,7 @@ export interface IChangeRequestChangeBase {
     };
 }
 
-export type PlausibleChangeRequestState =
+export type PlausibleChangeRequestPreviousState =
```

Comment:
As a further refactoring, if we move the previous state into the plausible context, I think we should also move this type definition to that file.